docs/bot-usage.md

```diff
@@ -2,6 +2,8 @@
 
 This page explains the different parameters of the bot and how to run it.
 
+Note: You can now use the bot by executing `source .env/bin/activate; freqtrade`.
+
```

Comment:
maybe something like 
```
!Note:
    If you've used setup.sh, don't forget to activate your virtual environment (`source .env/bin/activate`) before running freqtrade commands.
```

Note the syntax for notes, so it is displayed in the box in the documentation.

Reply:
Thanks for the feedback, updated below.